OAuthSwift/OAuthSwiftClient.swift

```diff
@@ -10,9 +10,10 @@ import Foundation
 
 var OAuthSwiftDataEncoding: NSStringEncoding = NSUTF8StringEncoding
 
-public class OAuthSwiftClient {
+public class OAuthSwiftClient: NSObject {
```

Comment:
Ok, thank you. 
It is my first significant contribution to a projet on Github so I do not have the right habits yet.